features/draft_environment.feature

```diff
@@ -11,14 +11,14 @@ Feature: Draft environment
     Then I should be on the case study page
     And the page should contain the draft watermark
 
-  @draft
+  @draft @high
```

Comment:
@danielroseman suggested that we could pick a document using content-register. That seems like a viable option although there's no option in content-register to limit the number of results it returns, so basically we'd need to query and get back a list of all 1200 case studies and then pick one (351kb, request takes 0.5 seconds). 

Let me know if you think this is worth doing @bradleywright?